static/src/javascripts/projects/common/modules/ui/toggles.js

```diff
@@ -0,0 +1,108 @@
+// @flow
+import mediator from 'lib/mediator';
+
+class Toggles {
+    component: ?HTMLElement;
+    controls: Array<HTMLElement>;
+
+    constructor(component: ?HTMLElement = document.body): void {
+        if (component) {
+            const controls = component.querySelectorAll('[data-toggle]');
+
+            this.component = component;
+            this.controls = [...controls];
+
+            mediator.on('module:clickstream:click', clickSpec => {
+                this.reset(clickSpec ? clickSpec.target : null);
+            });
+        }
+    }
+
+    init() {
+        this.controls.forEach(this.prepareControl, this);
+    }
```

Comment:
Don't worry, it's not you, it is superfluous. I moved the logic from `init` into the constructor when refactoring. After doing this eslint through an error as the `toggles` variable in 2/3 of the dependants was never used again...no further calls were made to it's api. I switched it back to use this `init` although I'd also considered adding an eslint-disable for those particular lines and getting rid of it. This was something I had actually intended to discuss so we can do that now! :smile: 

However as no subsequent calls are made to the new instances of 'Toggles' in it's dependants then I had an eslint error telling me